.github/CODEOWNERS

```diff
@@ -15,4 +15,12 @@
 # Global owners, will be the owners for everything in the repo. Membership is tracked via https://github.com/open-telemetry/community/blob/main/community-members.md 
 *   @open-telemetry/specs-semconv-maintainers @open-telemetry/specs-semconv-approvers
 
+# Schemas and schema file tooling
+/schemas/ @tigrannajaryan
+/internal/tools/schema_check.sh @tigrannajaryan
+
+# Logs semantic conventions
+/semantic_conventions/logs/ @tigrannajaryan
+/specification/logs/ @tigrannajaryan
```

Comment:
@jsuereth do we intend to have this folder in this repo or it'll be removed soon?

Reply:
I'd like to know when it is removed, so adding me as a codeowner even if temporarily is still useful.

Reply:
We can add you for now, but we're looking to remove it within 1-2 weeks in favor of the following kind of structure:
```
specification/
   registry.md
   resource/
   http/
   network/
   ...
```

I.e. HTTP would include ALL semconv (logs, metrics, traces) and the notion of sharing attributes will be easier to see.

Reply:
SGTM